src/modules/mavlink/mavlink_params.c

```diff
@@ -52,11 +52,10 @@ PARAM_DEFINE_INT32(MAV_COMP_ID, 1);
 /**
  * MAVLink protocol version
  * @group MAVLink
- * @value 0 Default to 1, switch to 2 if GCS sends version 2
- * @value 1 Always use version 1
- * @value 2 Always use version 2
+ * @value 1 v1 adapts to v2
```

Comment:
v1 adapts to v2?

Reply:
Short for whenever a v2 packet comes in it adapts the output protocol as well. Lack of a better word, does any native English speaker have a better idea? "Fallback" seems off.

Reply:
I called it "v1, auto-upgrade to v2"